src/xdocs/config_imports.xml

```diff
@@ -1291,7 +1291,8 @@ import java.util.stream.IntStream;
           </tr>
           <tr>
             <td>ordered</td>
-            <td>whether imports within group should be sorted</td>
+            <td>whether imports within group should be sorted
+              (It don't affect sorting for static imports.)</td>
```

Comment:
`don't` => `doesn't`
This was not done from a previous item.

Reply:
Done.